frontend/src/views/ProducerForm/ProductStep.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <h2 class="fr-h6">
+    <v-icon class="mr-1" name="ri-price-tag-2-fill" />
+    Dénomination commerciale
+  </h2>
+  <div class="grid grid-cols-2 gap-4">
+    <div class="col-span-2 md:col-span-1 max-w-md">
+      <DsfrInputGroup>
+        <DsfrInput :model="payload.name" label-visible label="Nom du produit" :required="true" />
+      </DsfrInputGroup>
+      <DsfrInputGroup>
+        <DsfrInput :model="payload.brand" label-visible label="Marque" />
+      </DsfrInputGroup>
+    </div>
+    <div class="col-span-2 md:col-span-1 max-w-md">
+      <!-- Useless? -->
```

Comment:
Je ne sais pas si ce commentaire "Useless" veut dire la même chose pour moi. Mais je me suis justement notée que "la gamme et l'arôme ne semblent pas vraiment fondamentaux"
Il faudrait surement lister sur un doc les différentes questions métier qui arrivent pour les poser en standup métier ?

Reply:
On a pensé la même chose (tu l'exprimes mieux que moi ahah). Ce n'est peut-être pas le plus urgent (car pire cas on aura des champs qui existent déjà aujourd'hui), mais ça serait top si on peut enlever du bruit pour l'user :+1:

Reply:
Je commence ça ici https://www.notion.so/betagouv-dinum/Piste-d-am-lioration-formulaire-de-d-claration-539ece46b7e84e949f89d318f53b7ad5